package.json

```diff
@@ -38,7 +40,7 @@
     "angular:build": "npm-run-all angular:build:ngpackagr angular:build:package",
     "angular:build:package": "cpy ./src/clr-angular/README.md ./dist/clr-angular/; replace '@VERSION' $npm_package_version ./dist/clr-angular/package.json",
     "angular:build:ngpackagr": "ng build clr-angular",
-    "angular:test": "ng test clr-angular --watch false --code-coverage && npm run angular:golden:test",
```

Comment:
We are losing the golden test here, we can't lose this or we will not be enforcing tracking changes of golden file.

Reply:
Sorry this is my going back and forth trying to narrow down what is using the most memory. Will be adding it back